src/main/java/edu/greenblitz/tobyDetermined/subsystems/swerve/KazaSwerveModule.java

```diff
@@ -140,15 +143,17 @@ public SwerveModuleState getModuleState() {
 	}
 	
 	@Override
-	public boolean isAtAngle(double targetAngleInRads, double errorInRads) {
-		double currentAngleInRads = getModuleAngle() % Math.PI;
-		targetAngleInRads = targetAngleInRads % Math.PI;
-		boolean isInRange = false;
-		for (int i = -1; i <= 1 ; i++) {
-			isInRange |= (currentAngleInRads +Math.PI*i < targetAngleInRads + errorInRads
-					&& currentAngleInRads +Math.PI*i > targetAngleInRads - errorInRads);
-		}
-		return isInRange;
+	public boolean isAtAngle(double targetAngleInRads, double tolerance) {
+		double currentAngleInRads = getModuleAngle();
+		boolean res = GBMath.modulo((currentAngleInRads - targetAngleInRads), (2 * Math.PI)) < tolerance
```

Comment:
What is res?

Reply:
Also, isn't the way the code is written beating the purpose of GB modulo?
I don't like that we use it, but if we use it, at least use it when it is useful. (And again, you know my opinion, this function should have a better name if used, etc...)